### PR TITLE
fix for #11 #12

### PR DESCRIPTION
**Pull request by kemcho**
redirect was not working because of loading info from state not from props, also fixed console errors on add transaction page.

### CODE REVIEW

File: client/src/components/portfolio/CurrentPortfolio.js [hunk 1]
```diff
@@ -24,15 +24,17 @@ class CurrentPortfolio extends Component {
     if (isEmpty(transactions)) {
       content = "Loading your current portfolio ...";
     } else {
-      const currentTransactions = this.state.transactions.map(transaction => (
```
Comment (raxitcholera) [inline client/src/components/portfolio/CurrentPortfolio.js:27]: as this was depending on the state and the state would not be created, when the history push happens, caused the issue?

File: client/src/components/portfolio/AddTransaction.js [hunk 2]
```diff
@@ -84,9 +84,9 @@ class AddTransaction extends Component {
                 <TextFieldGroup
                   name="date"
                   type="date"
-                  value={this.state.from}
```
Comment (raxitcholera) [inline client/src/components/portfolio/AddTransaction.js:87]: this could have been avoided..

File: client/src/actions/portfolioActions.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const addTransaction = (expData, history) => dispatch => {
     .catch(err =>
       dispatch({
         type: GET_ERRORS,
-        payload: err.response.data
```
Comment (raxitcholera) [inline client/src/actions/portfolioActions.js]: why should you not send out the error, if it did come up??

File: client/src/components/dashboard/Dashboard2.js [hunk 4]
```diff
@@ -1,59 +1,59 @@
-import React, { Component } from 'react';
```
Comment (raxitcholera) [inline client/src/components/dashboard/Dashboard2.js]: we need to have the same formatting setting or else this is going to keep happening!

File: client/src/components/portfolio/CurrentPortfolio.js (hunk 1 above)
Comment (kemcho) [reply client/src/components/portfolio/CurrentPortfolio.js:27]: yes

File: client/src/components/portfolio/AddTransaction.js (hunk 2 above)
Comment (kemcho) [reply client/src/components/portfolio/AddTransaction.js:87]: yes

File: client/src/actions/portfolioActions.js (hunk 3 above)
Comment (kemcho) [reply client/src/actions/portfolioActions.js]: good point, will resolve in next commit

**Review by raxitcholera — APPROVED** [review]
Looks good to me :)

**Pull request merged**